.eslintrc.json

```diff
@@ -22,5 +22,8 @@
         "react"
     ],
     "rules": {
+        "import/no-unresolved": "off",
+        "react/prefer-stateless-function": "off",
+        "react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }]
```

Comment:
Please don't change the ESlint config. If you have to do it on the source code.

Reply:
I was following this [link](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-filename-extension.md). And I wanted to allow both jsx and js. Cause my target is to separate the logic portion from it.

Reply:
First, let's make sure we using this config file: https://github.com/microverseinc/linters-config/blob/master/react-redux/.eslintrc.json

If you have to disable a rule use the option that disable the rule for specific files, blocks of code.

See https://eslint.org/docs/user-guide/configuring.html#configuring-rules

Reply:
- [Optional] You can use a function component in this milestone since `state` is not implemented yet.
- [Optional] You can also rename files with `jsx` syntax to `.jsx`

Reply:
> First, let's make sure we using this config file: https://github.com/microverseinc/linters-config/blob/master/react-redux/.eslintrc.json
>

What is the difference between my configuration and the link you provided one? except a 
 
> If you have to disable a rule use the option that disable the rule for specific files, blocks of code.
> 
> See https://eslint.org/docs/user-guide/configuring.html#configuring-rules

In [here](https://eslint.org/docs/user-guide/configuring.html#disabling-rules-only-for-a-group-of-files) I can see the disabling rules like i did. cause the rules I have added that is needed all over the project.

Reply:
> * [Optional] You can also rename files with `jsx` syntax to `.jsx`

I don't have a jsx file. So what do you mean by that, is confusing.

Reply:
> What is the difference between my configuration and your provided one? except a
rule.

Please note that it's not a configuration I provided. See **Remember to** section in the pathwright: https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896085/
> [Set up a code linter.](https://github.com/microverseinc/linters-config/tree/master/react-redux)